com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraEditor.cs

```diff
@@ -89,14 +90,24 @@ public void RemoveComponent(Camera camera, IEnumerable<Component> dependencies)
         [MenuItem("CONTEXT/Camera/Reset", false, 0)]
         static void ResetCamera(MenuCommand menuCommand)
         {
-            GameObject go = ((Camera)menuCommand.context).gameObject;
+            // Grab the current HDRP asset, we should not be executing this code if HDRP is null
+            var hdrp = (RenderPipelineManager.currentPipeline as HDRenderPipeline);
+            if (hdrp == null)
+                return;
```

Comment:
I think we can have a better workflow here.

If there is HDRP package but no HDAdditionalCameraData sibling, we do nothing with your solution. But basically the user want here to reset its camera that is not a HDRP one no?

Also if you have an HDAdditionalCameraData but the current SRP is not the HDRP one, a built-in Camera should be displayed. But this camera Reset contextual menu command should still perform the reset.

So I think, if I not mistaken that we should call camera.Reset(); before returning.